app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+
+def copy_file(command: str) -> None:
+    new_str = command.split(" ", 2)
+    cmd, file_first, file_second = new_str
+
+    if len(new_str) == 3 and file_first != file_second and cmd == "cp":
```

Comment:
you need to check `len(new_str)` before you split otherwise program may crash during unpacking